firmware/watchguard_sysa_dl.ksy

```diff
@@ -0,0 +1,137 @@
+meta:
+  id: watchguard_sysa_dl
+  title: Watchguard Fireware web update (Sysa-dl) file format
+  file-extension: sysa-dl
```

Comment:
BTW, `.wgpkg-dl` files used for AP devices use this format too (but `.wgpkg-dl` files apparently use only `info`, `WGPKG`, `SIGN` and `HMAC` sections, at least the following ones):

```
WatchGuard_AP/AP420
	https://cdn.watchguard.com/SoftwareCenter/Files/WAP/11_0_0/wap_AP420_11_0_0-36-4.zip	36.4MiB
		wap-AP420-11.0.0-36-4-all.wgpkg-dl
WatchGuard_AP/AP327X
	https://cdn.watchguard.com/SoftwareCenter/Files/WAP/11_0_0/wap_AP327X_11_0_0-36-4.zip	28.1MiB
		wap-AP327X-11.0.0-36-4-all.wgpkg-dl
WatchGuard_AP/AP325
	https://cdn.watchguard.com/SoftwareCenter/Files/WAP/11_0_0/wap_AP325_11_0_0-36-4.zip	28.4MiB
		wap-AP325-11.0.0-36-4-all.wgpkg-dl
WatchGuard_AP/AP322
	https://cdn.watchguard.com/SoftwareCenter/Files/WAP/8_8_3/wap_AP322_8_8_3-12.zip	31.2MiB
		wap-AP322-8.8.3-12-all.wgpkg-dl
WatchGuard_AP/AP225W
	https://cdn.watchguard.com/SoftwareCenter/Files/WAP/11_0_0/wap_AP225W_11_0_0-36-4.zip	29.2MiB
		wap-AP225W-11.0.0-36-4-all.wgpkg-dl
WatchGuard_AP/AP125
	https://cdn.watchguard.com/SoftwareCenter/Files/WAP/11_0_0/wap_AP125_11_0_0-36-4.zip	27.3MiB
		wap-AP125-11.0.0-36-4-all.wgpkg-dl
WatchGuard_AP/AP120
	https://cdn.watchguard.com/SoftwareCenter/Files/WAP/8_8_3/wap_AP120_8_8_3-12.zip	31.0MiB
		wap-AP120-8.8.3-12-all.wgpkg-dl
Legacy_Devices/AP100_AP102
	https://cdn.watchguard.com/SoftwareCenter/Files/WAP/1_2_9_16/wap_AP100_1_2_9_16.zip	6.9MiB
		wap-AP100-1.2.9.16-all.wgpkg-dl
Legacy_Devices/AP200
	https://cdn.watchguard.com/SoftwareCenter/Files/WAP/1_2_9_16/wap_AP200_1_2_9_16.zip	7.0MiB
		wap-AP200-1.2.9.16-all.wgpkg-dl
Legacy_Devices/AP300
	https://cdn.watchguard.com/SoftwareCenter/Files/WAP/2_0_0_11/wap_AP300_2_0_0_11.zip	9.4MiB
		wap-AP300-2.0.0.11-all.wgpkg-dl
Legacy_Devices/AP320
	https://cdn.watchguard.com/SoftwareCenter/Files/WAP/8_8_3/wap_AP320_8_8_3-12.zip	31.2MiB
		wap-AP320-8.8.3-12-all.wgpkg-dl
```

Reply:
Should we mention this at file meta section?

Reply:
something like "Watchguard Fireware web update (Sysa-dl) and  Watchguard AP devices  (wgpkg-dl) file format"